docs/source/pages/using.rst

```diff
@@ -13,6 +13,98 @@ Using Macaron
 
 .. contents:: :local:
 
+----------------------------------------
+Analyzing an artifact with a PURL string
+----------------------------------------
+
+Macaron can analyze an artifact (and it dependencies) to determine its supply chain security posture. To analyze an artifact, you need to provide the PURL identifier of the artifact:
+
+ .. code-block::
+
+  pkg:<package_type>/<artifact_details>
+
+Where ``artifact_details`` varies based on the provided ``package_type``. Examples for those currently supported by Macaron are as follows:
+
+.. list-table:: Examples of PURL strings for artifacts.
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Package Type
+     - PURL String
+   * - Maven (Java)
+     - ``pkg:maven/org.apache.xmlgraphics/batik-anim@1.9.1``
+   * - PyPi (Python)
+     - ``pkg:pypi/django@1.11.1``
+   * - Cargo (Rust)
+     - ``pkg:cargo/rand@0.7.2``
+   * - NuGet (.Net)
+     - ``pkg:nuget/EnterpriseLibrary.Common@6.0.1304``
+   * - NPM (NodeJS)
+     - ``pkg:npm/%40angular/animation@12.3.1``
+
+For more detailed information on converting a given artifact into a PURL, see `PURL Specification <https://github.com/package-url/purl-spec/blob/master/PURL-SPECIFICATION.rst>`_ and `PURL Types <https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst>`_
+
+
+To run Macaron on an artifact, we use the following command:
+
+.. code-block:: shell
+
+  ./run_macaron.sh analyze -purl <artifact-purl>
+
+
+''''''''''''''''''''''''''''''''''''''
+Automated repository and commit finder
+''''''''''''''''''''''''''''''''''''''
+
+Macaron is capable of automatically determining the repository and exact commit that matches the given artifact. For repository URLs, this is achieved through examination of SCM meta data found within artifact POM files (for Java), or use of Google's Open Source Insights API (for other languages). For commits, Macaron will attempt to match repository tags with the artifact version being sought, thereby requiring that the repository supports and uses tags on commits that were used for releases.
+
+Macaron will try to discover the corresponding repository of an artifact by default unless it is provided as input as shown `later <#analyze-repo>`_. This feature is enabled by default. To disable, or configure its behaviour in other ways, a custom ``.ini`` file should be passed to Macaron during execution. See `How to change the default configuration <#change-config>`_ for more details.
```

Comment:
```suggestion
By default, Macaron will try to discover the corresponding repository of an artifact unless it is already provided as input (as shown `later <#analyze-repo>`_). To disable or otherwise configure this behaviour, or others, a custom ``.ini`` file should be passed to Macaron during execution. See `How to change the default configuration <#change-config>`_ for more details.
```

Reply:
See commit [728d753](https://github.com/oracle/macaron/pull/816/commits/728d753678ee948b303d9b2fb9c672f245ed7ee0)